projects/1.21/assets/alchemists-arsenal/alkhar/lang/zh_cn.json

```
@@ -0,0 +1,158 @@
+  "item.minecraft.lingering_potion.effect.longer_legs" : "滞留型跨步药水 II",
+  "item.minecraft.lingering_potion.effect.longest_legs" : "滞留型跨步药水 III",
+  "item.minecraft.potion.effect.long_arms" : "长臂药水",
+  "item.minecraft.potion.effect.long_long_arms" : "长臂药水",
+  "item.minecraft.potion.effect.longer_arms" : "长臂药水 II",
+  "item.minecraft.potion.effect.longest_arms" : "长臂药水 III",
+  "item.minecraft.tipped_arrow.effect.long_arms" : "长臂之箭",
+  "item.minecraft.tipped_arrow.effect.long_long_arms" : "长臂之箭",
+  "item.minecraft.tipped_arrow.effect.longer_arms" : "长臂之箭 II",
+  "item.minecraft.tipped_arrow.effect.longest_arms" : "长臂之箭 III",
+  "item.minecraft.splash_potion.effect.long_arms" : "喷溅型长臂药水",
+  "item.minecraft.splash_potion.effect.long_long_arms" : "喷溅型长臂药水",
+  "item.minecraft.splash_potion.effect.longer_arms" : "喷溅型长臂药水 II",
+  "item.minecraft.splash_potion.effect.longest_arms" : "喷溅型长臂药水 III",
+  "item.minecraft.lingering_potion.effect.long_arms" : "滞留型长臂药水",
+  "item.minecraft.lingering_potion.effect.long_long_arms" : "滞留型长臂药水",
+  "item.minecraft.lingering_potion.effect.longer_arms" : "滞留型长臂药水 II",
+  "item.minecraft.lingering_potion.effect.longest_arms" : "滞留型长臂药水 III",
+  "item.minecraft.potion.effect.shrinkage_minor" : "缩小药水",
+  "item.minecraft.potion.effect.long_shrinkage_minor" : "缩小药水",
+  "item.minecraft.potion.effect.shrinkage" : "缩小药水 II",
+  "item.minecraft.potion.effect.long_shrinkage" : "缩小药水 II",
+  "item.minecraft.potion.effect.shrinkage_maior" : "缩小药水 III",
+  "item.minecraft.potion.effect.long_shrinkage_maior" : "缩小药水 III",
+  "item.minecraft.splash_potion.effect.shrinkage_minor" : "喷溅型缩小药水",
+  "item.minecraft.splash_potion.effect.long_shrinkage_minor" : "喷溅型缩小药水",
+  "item.minecraft.splash_potion.effect.shrinkage" : "喷溅型缩小药水 II",
+  "item.minecraft.splash_potion.effect.long_shrinkage" : "喷溅型缩小药水 II",
+  "item.minecraft.splash_potion.effect.shrinkage_maior" : "喷溅型缩小药水 III",
+  "item.minecraft.splash_potion.effect.long_shrinkage_maior" : "喷溅型缩小药水 III",
+  "item.minecraft.lingering_potion.effect.shrinkage_minor" : "滞留型缩小药水",
+  "item.minecraft.lingering_potion.effect.long_shrinkage_minor" : "滞留型缩小药水",
+  "item.minecraft.lingering_potion.effect.shrinkage" : "滞留型缩小药水 II",
+  "item.minecraft.lingering_potion.effect.long_shrinkage" : "滞留型缩小药水 II",
+  "item.minecraft.lingering_potion.effect.shrinkage_maior" : "滞留型缩小药水 III",
+  "item.minecraft.lingering_potion.effect.long_shrinkage_maior" : "滞留型缩小药水 III",
+  "item.minecraft.tipped_arrow.effect.shrinkage_minor" : "缩小之箭",
+  "item.minecraft.tipped_arrow.effect.long_shrinkage_minor" : "缩小之箭",
+  "item.minecraft.tipped_arrow.effect.shrinkage" : "缩小之箭 II",
+  "item.minecraft.tipped_arrow.effect.long_shrinkage" : "缩小之箭 II",
+  "item.minecraft.tipped_arrow.effect.shrinkage_maior" : "缩小之箭 III",
+  "item.minecraft.tipped_arrow.effect.long_shrinkage_maior" : "缩小之箭 III",
+  "item.minecraft.potion.effect.engorgement_minor" : "变大药水",

```

```suggestion
  "item.minecraft.potion.effect.engorgement_minor" : "巨化药水",
```
或者
```suggestion
  "item.minecraft.potion.effect.engorgement_minor" : "膨胀药水",
```
比较好。取决于具体效果（我猜是让体型变大？）